src/main/java/com/alextherapeutics/diga/DigaApiClient.java

```diff
@@ -127,6 +127,16 @@ public DigaInvoiceResponse invoiceDiga(DigaInvoice invoice)
     return performDigaInvoicing(invoice, billingInformation, DigaProcessCode.BILLING);
   }
 
+  public DigaInvoiceResponse digaInvoiceCorrection(DigaCorrectionInvoice invoice)
+      throws DigaCodeValidationException, DigaXmlWriterException {
+    return this.invoiceDiga(invoice);
+  }
+
+  public DigaInvoiceResponse digaTestInvoiceCorrection(
```

Comment:
can we have some documentation on these two public methods please?

Reply:
Documentation added